packages/medusa/src/api/admin/exchanges/[id]/inbound/items/[action_id]/route.ts

```diff
@@ -0,0 +1,96 @@
+import {
```

Comment:
question: Why do we fetch and return `returns` in these endpoints when they are for exchanges?

Reply:
In practice, inbounds are returns